src/pages/dev/general-message-passing/overview.mdx

```diff
@@ -21,14 +21,52 @@ With GMP you can:
 #### At the source chain
 
 1. User (dApp) calls a `callContract` (or `callContractWithToken`) function on the Axelar Gateway contract to initiate a call. Once the call is initiated, the user can see its status at https://axelarscan.io/gmp/[txHash] or programmatically track it via the [AxelarJS SDK](/dev/axelarjs-sdk/tx-status-query-recovery#query-transaction-status-by-txhash).
-2. User prepays the relayer gas fee on the source chain to Axelar’s Gas Services contract.
+2. The user prepays the relayer gas fee on the source chain to Axelar's Gas Services contract.
```

Comment:
should we be more specific and say "the gas for the decentralized Axelar consensus, and the necessary transactions to approve and execute on the destination chain" (link to gas service docs)

Reply:
Updated